Nightly build fails after added MingW macros

After commit 0c03280516d4a45f6ce3fb08765d9cd5ff7d9441 the build fails with this error message:
```
In file included from src/audio/data.h:4,
                 from src/audio/data.c:4:
src/audio/internal.h:275:22: error: unknown type name 'ssize_t'
  275 |     /*0x13C, 0x144*/ ssize_t bankDmaRemaining;
      |                      ^~~~~~~
make: *** [Makefile:832: build/us_pc/src/audio/data.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from src/audio/effects.h:4,
                 from src/audio/effects.c:4:
src/audio/internal.h:275:22: error: unknown type name 'ssize_t'
  275 |     /*0x13C, 0x144*/ ssize_t bankDmaRemaining;
      |                      ^~~~~~~
In file included from src/audio/effects.c:5:
src/audio/load.h:49:67: error: unknown type name 'ssize_t'; did you mean 'size_t'?
   49 | void audio_dma_partial_copy_async(uintptr_t *devAddr, u8 **vAddr, ssize_t *remaining, OSMesgQueue *queue, OSIoMesg *mesg);
      |                                                                   ^~~~~~~
      |                                                                   size_t
make: *** [Makefile:832: build/us_pc/src/audio/effects.o] Error 1
```
System: Linux 64-bit
Compiler: gcc 10.1.0

--- a/include/PR/ultratypes.h
+++ b/include/PR/ultratypes.h
@@ -31,6 +31,7 @@ typedef double f64;
 
 #include <stddef.h>
 #include <stdint.h>
+#include <sys/types.h>
 #if defined(__MINGW32__) 
 #include <_mingw.h>
 #if !defined(__MINGW64_VERSION_MAJOR)

--- a/src/pc/gfx/gfx_pc.c
+++ b/src/pc/gfx/gfx_pc.c
@@ -40,6 +40,8 @@
 #define MAX_LIGHTS 2
 #define MAX_VERTICES 64
 
+#define CLOCK_MONOTONIC 0
+
 struct RGBA {
     uint8_t r, g, b, a;
 };
@@ -159,7 +161,6 @@ static struct GfxRenderingAPI *gfx_rapi;
 #include <_mingw.h>
 #if !defined(__MINGW64_VERSION_MAJOR)
 #include <windows.h>
-#define CLOCK_MONOTONIC 0
 //https://stackoverflow.com/questions/5404277/porting-clock-gettime-to-windows
 struct timespec { long tv_sec; long tv_nsec; };    //header part
 int clock_gettime(int arg, struct timespec *spec)      //C-file part

